.vuepress/process_main_docs.rb

```diff
@@ -53,13 +53,13 @@ def process_main_docs(docs_source_dir)
     puts ">>> Migrating the Tutorial section"
 
 
-    Dir.glob("#{docs_source_dir}/tutorials/beginner/*.md") { |path|
+    Dir.glob("#{docs_source_dir}/tutorials/getting_started/*.md") { |path|
         file = File.basename(path)
         puts " -> #{file}"
-        process_file("#{docs_source_dir}/tutorials/beginner", file, "docs/tutorial", "#{$docs_repo_root}/tutorials/beginner/#{file}")
+        process_file("#{docs_source_dir}/tutorials/getting_started", file, "docs/tutorial", "#{$docs_repo_root}/tutorials/getting_started/#{file}")
```

Comment:
Your link in the sidebar should link to `docs/tutorial` because it's where the tutorial files are copied ;)
(since there's only one tutorial)